1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -0,0 +1,10 @@
+
+export function fizzbuzz(n: number): string {
+    if (typeof n !== "number") throw new Error("not a number");
```

Comment:
The error message could be more descriptive, like "Please enter a number" or something similar.